docs/simple-gateway/index.html

```diff
@@ -549,7 +549,7 @@ <h1 id="deploying-a-simple-gateway">Deploying a simple Gateway</h1>
 deployed together by the same owner. This represents a similar kind of model
 used for Ingress. In this guide, a Gateway and HTTPRoute are deployed which
 match all HTTP traffic and directs it to a single Service named <code>foo-svc</code>. </p>
-<p><img alt="Simple Gateway" src="../single-service-gateway.png" /></p>
+<p><img alt="Simple Gateway" src="single-service-gateway.png" /></p>
```

Comment:
This doesn't seem right?

Reply:
The images are inside `docs` directory in the generated output. I've moved them in the `docs-src` directory only.

Reply:
It looks like this was moved in docs-src as well though?
<img width="586" alt="Screen Shot 2020-12-03 at 12 04 30 PM" src="https://user-images.githubusercontent.com/419322/101082121-d19fad00-355f-11eb-97ac-450f8166cbe0.png">

Reply:
My bad. Updated. Thanks for catching this.